bigquery/google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -388,20 +401,24 @@ def _tabledata_list_page_to_arrow(page, column_names, arrow_types):
     return pyarrow.RecordBatch.from_arrays(arrays, names=column_names)
 
 
-def download_arrow_tabledata_list(pages, schema):
+def download_arrow_tabledata_list(pages, bq_schema):
```

Comment:
Renaming here is safe, since `_pandas_helpers` module is considered private (i.e. an implementation detail), and the helper is only invoked with `schema` as a positional argument throughout the BigQuery library.

The primary reason behind renaming is to avoid shadowing the existing `schema` import near the top of the file:
```py
from google.cloud.bigquery import schema
```
Not sure why the linter was not complaining about that, though...

Reply:
👍 to Peter's comment. It's a private module so we're free to rename.